src/components/Scrollable/Scrollable.css.ts

```diff
@@ -8,6 +8,7 @@ export const ScrollableUI = styled('div')`
   box-sizing: border-box;
   display: flex;
   max-height: 100%;
+  height: ${props => (props.fadeSides ? '100%' : 'auto')};
```

Comment:
ah that's neat, I didn't know about that feature. 💯 _implement that right away in the SideNavigation component_

Reply:
I think this is my favourite feature of styled components, makes the js in css worth it imo!

Reply:
😍